### ISSUE TITLE
Sub-selection from DOM.select

### ISSUE THREAD

**Issue by ckknight** [cyclejs/cycle-dom#40]
I've been playing with nested dialogues and have found it would be handy to pass an already-instantiated `DOM.select('.my-custom-component')` to a sub-component, so that it can do its event checking within that scope.

My ideal API:

```js
DOM.select('.my-custom-component').select('.my-other-custom-component').events('click')
```

Which would essentially translate into:

```js
DOM.select('.my-custom-component .my-other-custom-component').events('click')
```

**Comment by staltz** [comment]
That's a valid alternative, I've thought of it once, but for now we are investigating even deeper solutions: https://github.com/cyclejs/cycle-core/issues/167

**Comment by staltz** [comment]
@ckknight this is now implemented in the auto-scope branch, and released as version 8.0.0-rc2. Soon we will release it 8.0.0.
